libs/@local/hash-validation/src/property_type.rs

```diff
@@ -211,32 +181,42 @@ where
     }
 }
 
-impl<V, P, S> Schema<V, P> for OneOfSchema<S>
+impl<P> Schema<PropertyWithMetadata, P> for OneOfSchema<PropertyValues>
```

Comment:
why is this now only working on `PropertyWithMetadata`? I am guessing it's for the fact that `oneOf` will be relaxed only for property types, but not others?

Reply:
We only have `oneOf` in property types, no need to support it for all types. This way, we know the concrete error type.